lgsf/councillors/scrapers.py

```diff
@@ -3,11 +3,11 @@
 from bs4 import BeautifulSoup
 from dateutil.parser import parse
 
-from lgsf.scrapers import ScraperBase
-from lgsf.councillors import CouncillorBase
+from lgsf.scrapers import ScraperBase, CodeCommitMixin
+from lgsf.councillors import CouncillorBase, json
 
 
-class BaseCouncillorScraper(ScraperBase):
+class BaseCouncillorScraper(CodeCommitMixin, ScraperBase):
```

Comment:
This feels a bit brittle, as the order matters here. But I guess that's just multiple inheritance.